test/composertest.py

```diff
@@ -64,12 +53,13 @@ def setUp(self):
                                 "--silent",
                                 "--unix-socket", "/run/weldr/api.socket",
                                 "http://localhost/api/status"]
-        r = subprocess.call(self.ssh_command + curl_command, stdout=open(os.devnull, 'w'))
+        r = subprocess.run(self.ssh_command + curl_command, stdout=subprocess.DEVNULL)  # pylint: disable=no-member
```

Comment:
We're running this under python3. There's no need for the `no-member` pylint rule.

Reply:
There's pylint running under Python2 (to inspect the source code) which also discovers these files and complains about them. Hence the no-member disable.